HIP/hip-583.md

```diff
@@ -2,228 +2,461 @@
+
+Each account has a system-assigned account ID of the form `shard.realm.num`, where each value is an 8-byte number.
+[HIP 32](https://hips.hedera.com/hip/hip-32) defines a flow where, when transferring HBAR to an account, the "sender"
+can send to an `alias` for the account instead of the account number, where the `alias` is an encoding of the public
+key of the account. If there is an account with that `alias`, then the transfer succeeds. If the account does not exist,
+then it will be auto-created with the public key of the account matching that of the `alias` and the transfer succeeds.
+
+With [HIP 482](https://hips.hedera.com/hip/hip-482), Hedera now supports an integration path with Ethereum-native
+wallets, where those wallets can talk with the JSON-RPC Relay, which talks to the Hedera network. These wallets work
+under the assumption that all accounts are identified by a 20-byte number. If a user is using an Ethereum-native
+wallet such as MetaMask, they cannot send hbars to a `shard.realm.num` account ID, they must send it to a 20-byte
+address.
+
+Every account in Hedera has an Ethereum compatible 20-byte `alias` (known as the "long zero" `alias` since it is
+prefixed by many zeros), and an Ethereum-native wallet using the JSON-RPC relay **can** use that to send value to
+existing accounts with no modification to the existing accounts. However, what if a user wants to transfer value
+(hbars or tokens) to an account that doesn't yet exist? The Ethereum-native wallet cannot use the Hedera-specific
+key encoding defined in `HIP-32` because it is Hedera-specific. Rather, they need support for a new type of alias.
+
+In Ethereum, addresses are derived from an `ECDSA secp256k1` public key. Transferring value from one account to another
+involves a crypto transfer from one derived account address to another, and the target account is "auto created"
+if it did not previously exist. The holder of the `ECDSA secp256k1` key that corresponds to that derived address can
+use that new account since any transaction they sign will be signed by that key. 
+
+This HIP allows _any_ new account on Hedera to have an `ECDSA` derived `alias`, compatible with Ethereum addresses, and
+permits the use of this `alias` for all operations supported by `alias` today. This HIP expands support in the
+auto-create flow for `ECDSA` derived addresses to be used as the `alias`. It also adds support to `CryptoCreate`
+transactions for creation with either a key-based `alias` or, an `ECDSA` derived `alias`.
+
+Rationale
+------------------------------------------------------------------------------------------------------------------------
+
+Fundamental in the design of the Hedera network are the concepts of _accounts_, account _identifiers_, and _keys_. These
+are fundamentally different entities. An `Account` must have a key. The key for the account may be a simple key
```

Comment:
I guess it is not even true then, because some accounts like `0.0.800` have no key at all. Maybe they aren't a full or complete account, but maybe that terminology isn't right, because it isn't hollow either. What if I just say:

> Today, a user `Account` must have a key

I wanted to avoid talking about the HIP proposal in the preamble, but just explain the situation today.